lib/common.js

```diff
@@ -24,6 +24,18 @@ function getConfig(configName) {
   return {};
 }
 
+function getOutputJson() {
```

Comment:
This function seems to be unique to Lighthouse. This `common.js` file was originally storing methods shared across Lighthouse and bbc-a11y, so I think it's home should be in `lib/lighthouse.js`

Reply:
Also, the name suggests it's getting JSON, but it's actually getting part of a file path.